content/c/concepts/memory-management/memory-management.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: 'Memory Management'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
```

Comment:
Please could we add a description here. Perhaps something like: 

Description: Memory management is the  process by which computer programs are assigned with physical or virtual memory space.
```

Reply:
ok sure.